### PR TITLE
PROGRAMMES-6655 Adding Touchcast Content Block

### PR DESCRIPTION
**Pull request by votemike**
https://jira.dev.bbc.co.uk/browse/PROGRAMMES-6655

### CODE REVIEW

File: src/Ds2013/Presenters/Domain/ContentBlock/Touchcast/touchcast.html.twig [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+{% block extraClasses %}
+    {% if not content_block.isInPrimaryColumn() %}
+        component--box-flushbody
+    {% endif %}
+{% endblock %}
+
+{% block blockContent %}
+    <div class="third-party third-party--touchcast">
+        <div class="third-party__embed">
+            {#<iframe src="//bbc-vip.touchcast.com/embed/{{ content_block.getBlock().getTouchcastId() }}?skincolor=<?=substr($this->branding->global->highlight_bg_color, 1)?>" frameborder="0" scrolling="no" allowfullscreen></iframe>#}
```
Comment (votemike) [inline src/Ds2013/Presenters/Domain/ContentBlock/Touchcast/touchcast.html.twig]: @BjornTwachtmann Do we _need_ the skincolor param here? It doesn't seem to make any difference. And I can't find any information on it online or on Confluence.

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/Ds2013/Presenters/Domain/ContentBlock/Touchcast/touchcast.html.twig (hunk 1 above)
Comment (BjornTwachtmann) [reply src/Ds2013/Presenters/Domain/ContentBlock/Touchcast/touchcast.html.twig]: I have no idea, happy to help you look into it, but I never worked on these

**Comment by Tasiobg** [comment]
Other block have test in ContentBlockMapperTest.php
It wont stop me from approving the PR  but just letting you know in case you see some value on it

**Comment by votemike** [comment]
@Tasiobg only some of them do. And I think this is such a simple block, there's no need.

**Pull request merged**